runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -138,6 +139,25 @@ sub stopDataManagerContainer {
 	my $logger         = get_logger("Weathervane::DataManager::AuctionKubernetesDataManager");
 	my $cluster = $self->host;
 	
+	# before deleting the deployment, capture info about state of the datamanage 
+	# pod and dump it to the log
+	my $namespace = $self->appInstance->namespace;
+	my $kubernetesConfigFile = $cluster->getParamValue('kubeconfigFile');
+	my $context = $cluster->getParamValue('kubeconfigContext');
+	my $contextString = "";
+	if ($context) {
+	  $contextString = "--context=$context";	
+	}
+	my $cmd = "kubectl describe pod --namespace=$namespace --selector=impl=auctiondatamanager --kubeconfig=$kubernetesConfigFile $contextString";
+    my ($cmdFailed, $outString) = runCmd($cmd);
```

Comment:
The start line formatting varies through this section down through line 176, if that matters.

Reply:
fixed